src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -408,6 +408,10 @@ private function getAutowiredReference(TypedReference $reference, bool $filterTy
                     }
                 }
             }
+
+            if ($reference->getAttributes() && !$this->container->has($name)) {
```

Comment:
I'm not sure about the `has()` check. Why is it needed?

Reply:
Fixed! was necessary on a previous version of my fix